packages/server/src/routes/members/trust.ts

```diff
@@ -50,11 +50,7 @@ export const trust = (
         created_at: firestore.FieldValue.serverTimestamp()
       };
       const newOperationRef = operationsCollection.doc();
-      if (memberToTrust.get("request_invite_from_member_id") === loggedInUid) {
```

Comment:
what happens to current members who have verified but not trusted? is that possible?